src/main/java/org/folio/circulation/resources/CheckInProcessAdapter.java

```diff
@@ -132,7 +133,11 @@
     if (firstRequest == null) {
       return completedFuture(succeeded(null));
     }
-    return servicePointRepository.getServicePointById(UUID.fromString(firstRequest.getPickupServicePointId()))
+    return servicePointRepository.getServicePointById(
```

Comment:
Does this deliberately try to request a service point with a null ID?

Could the code instead return eagerly if no service point is present?

Reply:
changed